src/ui/preferences-window/tabs/UpdatesTab.swift

```diff
@@ -24,9 +25,10 @@ class UpdatesTab: NSViewController, PreferencePane {
         policyLabel.isSelectable = false
         let policies = StackView([dontPeriodicallyCheck, periodicallyCheck, periodicallyInstall], .vertical)
         policies.spacing = GridView.interPadding / 2
+        checkForUpdates = NSButton(title: NSLocalizedString("Check for updates now…", comment: ""), target: nil, action: #selector(UpdatesTab.checkForUpdatesNow))
         let grid = GridView([
             [policyLabel, policies],
-            [NSButton(title: NSLocalizedString("Check for updates now…", comment: ""), target: nil, action: #selector(UpdatesTab.checkForUpdatesNow))],
+            [checkForUpdates],
```

Comment:
While I think this refactor is fine, I would rather avoid unrelated code in a PR. I suggest you either revert it, on separate it in a separate commit like `refactor: made buttons code more uniform`